control/autoware_control_toolbox/CMakeLists.txt

```diff
@@ -0,0 +1,176 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Let me ask you first whether there is any package in autoware.universe which will depend on the autoware_control_toolbox package after this PR is merged.
Will your nonlinear mpc depend on this package?

Reply:
Yes, but not only nonlinear MPC; we have delay compensation, adaptive controllers, and disturbance generator packages in the row. This ACT also makes designing controller, observer, and low-level control tasks easier.